src/features/purchases/screen/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { SafeAreaView, Text } from 'react-native';
+
+import { translate } from 'localization/hooks';
+
+import styles from './styles';
+
+const PurchasesScreen = () => {
+  return (
+    <SafeAreaView style={styles.container}>
+      <Text accessibilityRole="text" style={styles.title}>
+        {translate('screen.purchases.title')}
+      </Text>
+    </SafeAreaView>
+  );
+};
+
+export default PurchasesScreen;
```

Comment:
There is a long-living discussion about named vs default export. I know our codebase uses mostly default export, but worth the reading:
- https://www.freecodecamp.org/news/difference-between-default-and-named-exports-in-javascript/
- Some opinions: https://www.reddit.com/r/react/comments/105hs4q/whats_better_named_or_default_exports/

I'll always prefer named exports because it's easy to search, enforces consistency, etc, but that's totally fine either way